decentraland/unity-renderer
# Fix: avoid bug that duplicated animations in memory

## What does this PR change?

Avoids a duplication animations when being downloaded.

...

## How to test the changes?

1. In the editor launch the game
2. Take a memory snapshot
3. Perform various changes in the avatar editor
4. Take another snapshot
5. Verify that the animation clips are normal and there aren't duplicated ones (keep in mind that the default ones are always 2 for male and female)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
